utils/warnings.ml

```diff
@@ -170,6 +171,7 @@ let number = function
   | Redefining_unit _ -> 65
   | Unused_open_bang _ -> 66
   | Unused_functor_parameter _ -> 67
+  | Interface_not_found -> 68
 ;;
 
 let last_warning_number = 67
```

Comment:
You need to update the last warning number to 68 .

Reply:
Thanks it worked! However `stdlib/pervasives.ml` itself doesn't have an interface file and since make file treats warnings as error `-warn-error` it will not finish make operation and end up with an error.

One possible fix could be that we add a Clflags which is used internally while performing make operation to disable Interface check. Otherwise we can also create mli file for `stdlib/pervasives.ml`

Reply:
It would be much simpler to remove the warning from the default warnings.